linux/Dockerfile

```diff
@@ -9,8 +9,15 @@ RUN sudo curl -sLO https://github.com/PowerShell/PowerShell/releases/download/v7
 
 RUN sudo apt-get update \
   && sudo apt-get install -y \
-    gettext-base \
-  && sudo rm -rf /var/lib/apt/lists/*
+    gettext-base
+
+RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+  && sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+  && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
```

Comment:
I'd run these first (before line 10) and do the `apt-get update && apt-get install` once.